MapboxAndroidDemo/src/china/res/layout/activity_style_color_switcher.xml

```diff
@@ -1,31 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:mapbox="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
     tools:context=".examples.styles.ColorSwitcherActivity">
 
+
+    <com.mapbox.mapboxsdk.maps.MapView
```

Comment:
Does this example not need to use the `ChinaMapView`?

Reply:
it needs it. Added ✔️